adal/automation/WinFormsAutomationApp/MainForm.cs

```diff
@@ -99,6 +101,14 @@ private void acquireDeviceCode_Click(object sender, EventArgs e)
         {
             _commandToRun = AuthenticationHelper.AcquireDeviceCodeAsync;
             pageControl1.SelectedTab = dataInputPage;
-        }        
+        }
+
+        private void writeToFile_Click(object sender, EventArgs e)
+        {
+            if (File.Exists(AuthResultFile))
```

Comment:
>AuthResultFile [](start = 28, length = 14)

want to add it to the location of the running process? Not sure this is what File.Write does by default. Consider making the location of the file explicit.

Reply:
i verified that the default behavior is to use the current running process location which will make it easier for the agent to find it.